src/ranged.cpp

```diff
@@ -283,7 +293,8 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
             // Critter can still dodge the projectile
             // In this case hit_critter won't be set
             if( attack.hit_critter != nullptr ) {
-                splatter( blood_traj, dealt_dam.total_damage(), critter );
+                g->m.add_splatter_trail( critter->bloodType(), blood_traj,
+                                         blood_trail_len( dealt_dam.total_damage() ) );
```

Comment:
This doesn't seem to do anything, I'm not getting any blood splatters from headshotting zeds with .50.